src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -5274,10 +5274,10 @@
      * |   |   `--IDENT -&gt; String
      * |   |--IDENT -&gt; hello
      * |   `--ASSIGN -&gt; =
-     * |       `--EXPR -&gt; EXPR
```

Comment:
it seems idents were correct, please double check with ast fragment you provided

```
 |   `--ASSIGN -> =
 |       `--EXPR -> EXPR
 |           `--TEXT_BLOCK_LITERAL_BEGIN -> """
 |               |--TEXT_BLOCK_CONTENT -> \n                Hello, world!\n
 |               `--TEXT_BLOCK_LITERAL_END -> """
 |--SEMI -> ;
```

Reply:
The idents were correct but spacing was not correct and also the previous person committed it and he closed it and maybe it made chaos. Still, can you help me to resolve this @timurt ?